### ISSUE TITLE
iphone and ipad for android

### ISSUE THREAD

**Issue by badpenguin** [ripple-emulator/ripple#13]
Can you please add some android devices with the same screen as the iphone and ipad?
Would be nice to have them for people without IOS for testing the screen size.

Otherwise would be nice to add have the feature to add custom screen size
or that ripple reads a config file.

**Comment by TimBarham** [comment]
Hey @badpenguin - can't you get the same effect just by adding the `ios` platform to your Cordova project, then selecting an iPad in Ripple?

**Comment by badpenguin** [comment]
@TimBarham nope, it does not works if you're on linux, it only works if you have a mac.
I already did it "in a raw way" by hacking the minified ripple.js in node_modules.

**Comment by TimBarham** [comment]
@badpenguin - when you say "does not work", what exactly *does* happen?

I've tried this on Windows and it works fine there. I'm not aware of any reason it shouldn't work on Linux.

**Comment by badpenguin** [comment]
@TimBarham thanks! Maybe something changed in cordova, in the past i was unsable to add the IOS platform to linux, now it works (i just get a warning); then ripple works :)

BTW it still be nice to have the change to add custom display size when future devices come out (mobile world change each 3 months).
